src/kirby/components/modal/action-sheet/action-sheet.component.spec.ts

```diff
@@ -55,6 +56,7 @@ describe('ActionSheetComponent', () => {
         SpinnerComponent,
         InfiniteScrollDirective,
         ButtonComponent,
+        ListItemColorDirective,
```

Comment:
Could you please use `MockDirective` from `ng-mocks`.
```suggestion
        MockDirective(ListItemColorDirective),
```

Then we also do real unit testing 🚀

Reply:
But do we really want that here? All the other dependencies are included without mock. Maybe it also makes sense to keep this as much of an integration test as possible